prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -438,7 +439,8 @@
       <mat-icon>edit</mat-icon>
     </button>
   </app-page-subheader>
-  <app-enrollee-property [title]="selfDeclarationQuestions[SelfDeclarationTypeEnum.HAS_REGISTRATION_SUSPENDED]"
+  <app-enrollee-property *ngIf="admin && hasRegistrationSuspended() === true || !admin"
```

Comment:
I would always suggest enforcing precedence using brackets for two reasons. That one time that you don't and you should have will cost you hours of debugging (definitely speaking from experience), and the second is for readability as the brackets similar to ternaries provide a way to increase your ability to scan and take in the pertinent information.